utilities/rocAL/rocAL_unittests/pixel_comparison/image_comparison.py

```diff
@@ -0,0 +1,147 @@
+from PIL import Image
+import os
+import sys
+import datetime
+import logging
+
+def compare_pixels(img1, img2, aug_name, width, height, image_offset = 0):
+    pixel_difference = [0,0,0,0,0,0]
+    if'rgb' in aug_name:
+        pixels1 = img1.load()
+        pixels2 = img2.load()
+        total_valid_pixel_count = width * height * 3
+        for wt in range(width):
+            for ht in range(height):
+                ht = ht + image_offset
+                if pixels1[wt,ht] != pixels2[wt,ht]:
+                    r = abs(pixels1[wt,ht][0]-pixels2[wt,ht][0])
+                    g = abs(pixels1[wt,ht][1]-pixels2[wt,ht][1])
+                    b = abs(pixels1[wt,ht][2]-pixels2[wt,ht][2])
+                    if(r > 4):
+                        r = 5
+                    if(g > 4):
+                        g = 5
+                    if(b > 4):
+                        b = 5
+                    pixel_difference[r] += 1
```

Comment:
we might want to capture larger pixel differences as well. Please incoorporate in future PR